main.py

```diff
@@ -0,0 +1,55 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def get_authors(db: Session = Depends(get_db), limit: int = 10, skip: int = 0):
+    return crud.get_all_authors(db=db, limit=limit, skip=skip)
+
+
+@app.get("/authors/{pk}", response_model=schemas.Author)
+def get_author(pk: int, db: Session = Depends(get_db)):
+    return crud.get_author_by_id(db=db, author_id=pk)
```

Comment:
Consider adding error handling for the case where the author is not found. You can raise an HTTPException with a 404 status code if `crud.get_author_by_id` returns `None`.